test/tbb/test_parallel_for_each.cpp

```diff
@@ -127,10 +127,12 @@ concept can_call_parallel_for_each_with_cbs = requires( ContainerBasedSequence c
     tbb::parallel_for_each(const_cbs, body, ctx);
 };
 
+using CorrectCBS = test_concepts::container_based_sequence::Correct;
+
 template <typename Body>
 concept can_call_parallel_for_each =
-    can_call_parallel_for_each_with_iterator<test_concepts::container_based_sequence::iterator, Body> &&
-    can_call_parallel_for_each_with_cbs<test_concepts::container_based_sequence::Correct, Body>;
+    can_call_parallel_for_each_with_iterator<CorrectCBS::iterator, Body> &&
```

Comment:
May be add `using namespace test_concepts::container_based_sequence`,  simply use `Correct` and remove `using`?

Reply:
I think that the CorrectCBS name better describes that this is a container-based sequence argument.